lore/pipelines/holdout.py~

```diff
@@ -0,0 +1,333 @@
+from __future__ import absolute_import
```

Comment:
@jeremystan Is this a tempfile created by you editor that should be removed?

Reply:
@montanalow yes, I yanked it